vboxwebsrv/vboxwebsrv.go

```diff
@@ -15800,7 +15800,7 @@ type IAdditionsFacility struct {
 }
 
 type IMediumAttachment struct {
-	XMLName xml.Name `xml:"http://www.virtualbox.org/ IMediumAttachment"`
+	// XMLName xml.Name `xml:"http://www.virtualbox.org/ IMediumAttachment"`
```

Comment:
Yup agree with your comment above for loglevel.  This is necessary due to an error that was being returned.  It was throwing a returnval error which seemed aligned to not parsing the information correctly.